test_common/KokkosKernels_TestUtils.hpp

```diff
@@ -620,35 +620,35 @@ class RandCscMat {
  public:
   std::string info;
   /// Constructs a random csc matrix.
-  /// \param m The number of rows.
-  /// \param n The number of columns.
+  /// \param nvecs The number of vecs.
```

Comment:
Same as above. Not resolved from last review.

Reply:
Same as above.